### ISSUE TITLE
Chaturbate 

### ISSUE THREAD

**Issue by Haddonfieldelm** [Colossal1/plugin.video.xxx-o-dus#34]
Still getting the error "failed to get any playable link".on all thenperformers.
I appreciate all that you do 😊

**Comment by Biz387** [comment]
It may need a new version uploaded.. be patient

**Comment by Haddonfieldelm** [comment]
Ok thanks 😊On Nov 12, 2017 6:04 PM, Biz387 <notifications@github.com> wrote:It may need a new version uploaded.. be patient

**Comment by Colossal1** [comment]
I pushed an update out last night - version 3.00.033

Please check you are on that version and try chaturbate again.

Let me know the results so I can close the issue.

Thanks

**Comment by Haddonfieldelm** [comment]
Yes that worked . Thanks so much 😊On Nov 13, 2017 1:24 PM, Team Colossus <notifications@github.com> wrote:I pushed an update out last night - version 3.00.033
Please check you are on that version and try chaturbate again.
Let me know the results so I can close the issue.
Thanks
